include/aspect/utilities.h

```diff
@@ -95,7 +95,24 @@ namespace aspect
      */
     template <int dim>
     Point<dim>
-    cartesian_coordinates(const std_cxx11::array<double,dim> &scoord);
+    spherical_to_cartesian_coordinates(const std_cxx11::array<double,dim> &scoord);
+
+    /**
+     * Returns ellispoidal coordinates of a cartesian point. The returned array
```

Comment:
But I think it's ok to leave the actual function name in lowercase. I'm only talking about the documentation.